PackageExplorer/MefServices/PluginManager.cs

```diff
@@ -90,7 +100,7 @@ public PluginInfo AddPlugin(IPackage plugin)
                         {
                             Directory.Delete(targetPath);
                             UIServices.Value.Show(
-                                "Adding plugin failed. The selected package does not have any assembly inside the 'lib\\net40' folder.",
+                                "Adding plugin failed. The selected package does not have any assembly inside the 'lib\\net*' folder.",
```

Comment:
I think it would be nice to list all "searched" directories.

Reply:
I prefer `”lib\\net40', 'lib\\net45', 'lib\\net451',`

O:)

Reply:
Oops I type to slow

Reply:
I can create a new PR if you want 😄